Exiled.API/Extensions/CommonExtensions.cs

```diff
@@ -61,5 +61,26 @@ public static AnimationCurve Add(this AnimationCurve curve, float amount)
 
             return curve;
         }
+
+        /// <summary>
+        /// Get a chance of a number falling out (as a percentage).
+        /// </summary>
+        /// <param name="chance">The value of chance.</param>
+        /// <returns>Whether the necessary chance has fallen out.</returns>
+        public static bool Chance(this double chance) => Chance(chance);
```

Comment:
this isn't double check but a float check i would prefer than you use Exiled.Loader.Random.NextDouble() so you will compare it with double and not use float

Reply:
I have updated my solution. Maybe it will be better